vertical-pod-autoscaler/e2e/v1/autoscaling_utils.go

```diff
@@ -65,18 +65,16 @@ const (
 )
 
 var (
-	resourceConsumerImage           = imageutils.GetE2EImage(imageutils.ResourceConsumer)
-	resourceConsumerControllerImage = imageutils.GetE2EImage(imageutils.ResourceController)
+	resourceConsumerImage = imageutils.GetE2EImage(imageutils.ResourceConsumer)
 )
 
 var (
-	// KindRC var
+	// KindRC is the GVK for ReplicationController
```

Comment:
nit: missing dot at the end of the sentence. The same goes for other comments.

Reply:
This file is mostly copied from k8s repo. I'm not entirely sure we need this copy, but I'd rather not mix it with this PR. The same goes for the other comment.

Reply:
ack.